.github/workflows/test-native.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           ccache -s
           cd ${GITHUB_WORKSPACE}/presto-native-execution
           make velox-submodule
-          cmake -B _build/debug -GNinja -DTREAT_WARNINGS_AS_ERRORS=1 -DENABLE_ALL_WARNINGS=1 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+          cmake -B _build/debug -GNinja -DTREAT_WARNINGS_AS_ERRORS=1 -DENABLE_ALL_WARNINGS=1 -DCMAKE_BUILD_TYPE=Debug -DPRESTO_ENABLE_PARQUET=ON -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
Please separate this change in its own PR, since that is orthogonal to the changes here.

Reply:
I opened https://github.com/prestodb/presto/pull/19476 to address this.